smack-extensions/src/main/java/org/jivesoftware/smackx/pubsub/Node.java

```diff
@@ -224,6 +224,31 @@ public DiscoverInfo discoverInfo() throws NoResponseException, XMPPErrorExceptio
         return subElem.getSubscriptions();
     }
 
+    /**
+     * Modify the subscriptions for this PubSub node as owner.
+     * <p>
+     * Note that the subscriptions are _not_ checked against the existing subscriptions
+     * since these are not cached (and indeed could change asynchronously)
+     * </p>
+     *
+     * @param changedSubs subscriptions that have changed
+     * @return <code>null</code> or a PubSub stanza with additional information on success.
+     * @throws NoResponseException
+     * @throws XMPPErrorException
+     * @throws NotConnectedException
+     * @throws InterruptedException
+     * @see <a href="https://xmpp.org/extensions/xep-0060.html#owner-subscriptions-modify">XEP-60 § 8.8.2 Modify Subscriptions</a>
+     * @since 4.2
```

Comment:
Since you are targeting `master`, this should probably be `@since 4.3`.

Reply:
or 4.4 ?! (4.3 is out already, right ? 4.3.X ?)

Reply:
Smack 4.2 is currently stable Smack version. 4.3 is not yet released and is the current 'master' branch (although I'll branch out 4.3 soon™). So @vanitasvitae is right, it should say `@since 4.3`.